clients/venice-client/src/main/java/com/linkedin/venice/fastclient/DelegatingAvroStoreClient.java

```diff
@@ -26,7 +60,6 @@ protected CompletableFuture<V> get(GetRequestContext requestContext, K key) thro
     return delegate.get(requestContext, key);
   }
 
-  @Override
```

Comment:
Why remove this?

Reply:
Thanks for finding it. Fixed it, should have been removed by mistake.